IOPool/Streamer/src/StreamerInputFile.cc

```diff
@@ -257,7 +298,7 @@ namespace edm {
       }
       eventRead = true;
       if (eventSkipperByID_) {
-        EventHeader* evh = (EventHeader*)(&eventBuf_[0]);
+        EventHeader* evh = (EventHeader*)(eventPtr);
```

Comment:
Is this guaranteed to have the correct byte alignment now?

Reply:
Here eventPtr isn't changed (using zeroCopy=false for reading the event header in previous step) and still points to the beginning of the std::vector allocation, so it should be ok.

Reply:
Now I see that there is the same problem with the EventMsgView:
```
uint32 EventMsgView::protocolVersion() const {
  EventHeader* h = (EventHeader*)buf_;
  return h->protocolVersion_;
}
```
so it's not safe to use non-aligned starting address and we need to copy the event....
It's still safe for the InitMessage, because it's always at the beginning of the buffer.
For event it could be enabled if event size padding is introduced to the streamer format.